src/app/components/experiment-workspace/experiment-workspace.component.ts

```diff
@@ -207,7 +212,7 @@ export class ExperimentWorkspaceComponent implements OnInit, OnDestroy {
             }
         }
         this.activePlugin = plugin;
-        this.frontendUrl = frontendUrl;
+        this.frontendUrl = frontendUrl + `?plugin-endpoint-url=${plugin.url}`;
```

Comment:
If the frontend URL should include this param, then the plugin can just include the param in the entry point uiHref. There is no need to change the QHAna UI for this!